parasolr/query/aliased_queryset.py

```diff
@@ -105,6 +105,9 @@ def get_facets(self) -> Dict[str, int]:
         '''Extend :meth:`parasolr.query.queryset.SolrQuerySet.get_facets`
         to use aliased field names for facet and range facet keys.'''
         facets = super().get_facets()
+        # bail out if empty dict is returned
+        if not facets:
+            return facets
```

Comment:
maybe it isn't a python convention, but this would feel more explicit if it was just `return None` - even though it's obviously the same. i think my brain just sees `return [var]` and implicitly assumes `var` must be "some value" if it's `var`, rather than `None`...if that makes sense

Reply:
ah, but I guess this is the empty list, which is falsy...and maybe `return []` isn't as clear. never mind